src/views/characters/CharactersList.vue

```diff
@@ -1,5 +1,5 @@
 <script setup>

```

Comment:
Your changes are great! 
I just have noticed some other things in this file.

- Can you set `:details='Aproved'`  to `characterStore.characterItem.approved ? 'Approved': 'Not approved'`
![image](https://github.com/user-attachments/assets/2fa27f3c-1837-4d99-8bce-f2b0cc5d28e0)
- And can you set the text `Zaken aan het laden` to `Karakters aan het laden`
![image](https://github.com/user-attachments/assets/eedc5030-a07d-41f3-9585-4fe17dcac006)